pkgs/applications/science/biology/trimmomatic/default.nix

```
@@ -0,0 +1,57 @@
+{ ant
+, fetchFromGitHub
+, jdk11_headless
+, jdk8_headless
+, jre_minimal
+, lib
+, makeWrapper
+, stdenv
+}:
+let
+  my_jre = jre_minimal.override {
+    modules = [ "java.base" "java.logging" ];
+    jdk = jdk11_headless;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "trimmomatic";
+  version = "0.39";
+
+  src = fetchFromGitHub {
+    owner = "usadellab";
+    repo = "Trimmomatic";
+    rev = "v" + version;
+    sha256 = "sha256-u+ubmacwPy/vsEi0YQCv0fTnVDesQvqeQDEwCbS8M6I=";
+  };
+
+  nativeBuildInputs = [ jdk8_headless ant makeWrapper ];
+
+  buildPhase = "ant";
```

Have you ever read any Nixpkgs code before?

```suggestion
  buildPhase = ''
    runHook preBuild
    
    ant
    
    runHook postBuild
  '';
```